### PR TITLE
Nw parks injection

### PR DESCRIPTION
**Pull request by wesseln1**

### CODE REVIEW

**Comment by wesseln1** [comment]
got injection into dom working as well as my functions to retrieve name and location of park

**Comment by wesseln1** [comment]
git checkout -b nw-parks-injection
git pull origin nw-parks-injection

File: src/scripts/parks/parks_fetch.js [hunk 1]
```diff
@@ -19,18 +12,28 @@ const parkSearch = function (eventListenerValue) {
...
 const parkFetch = function (parkValue) {
     fetch(`https://data.nashville.gov/resource/xbru-cfzi.json?${parkValue}`, {
         headers: {
             "Accept": "application/json",
             "X-App-Token": "T7zn5kO3GnGyvLhoGQjL1b2EN"
         }
     })
-        .then(data => data.json())
-        .then((data) => {
-            console.log(data);
```
Comment (CurtainUp) [inline src/scripts/parks/parks_fetch.js]: Get outta heeeeeere

File: src/scripts/parks/parks_filterdata.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+function accessURL (parkar) {
+    parkar.forEach(parks => {
+        parkNames.push(parks.url)
+    })
+    console.log(parkNames)
```
Comment (CurtainUp) [inline src/scripts/parks/parks_filterdata.js]: get outta heeeeere

**Comment by jasehackman** [comment]
Add park_filterdata.js to index

**Comment by jasehackman** [comment]
#Crushedit!

**Comment by CurtainUp** [comment]
🐱‍🐉

**Pull request merged**